dataland-frontend/tests/e2e/fixtures/common/CountryFixtures.ts

```diff
@@ -1,20 +1,12 @@
-import { faker } from "@faker-js/faker";
+import { getAllCountryCodes } from "@/utils/CountryCodeConverter";
 
-/**
- * Randomly returns a country from "BE", "DE", "FR", "IT", "LU", "PL", "PT", "ES"
- * @returns the randomly chosen country code
- */
-export function generateIso2CountryCode(): string {
-  const someCommonIso2CountryCodes = ["BE", "DE", "FR", "IT", "LU", "PL", "PT", "ES"];
-  return someCommonIso2CountryCodes[Math.floor(Math.random() * someCommonIso2CountryCodes.length)];
-}
+const allIso2CountryCodes = getAllCountryCodes();
 
 /**
- * Randomly returns a list of country codes
- * @returns the randomly generated list of country codes
+ * Randomly returns one Iso2 country code from all available Iso2 country codes
+ * @returns the randomly chosen country code
  */
-export function generateListOfIso2CountryCodes(): string[] {
-  return Array.from({ length: faker.number.int({ min: 1, max: 5 }) }, () => {
-    return generateIso2CountryCode();
-  }).sort((a, b) => a.localeCompare(b));
+export function getRandomIso2CountryCode(): string {
```

Comment:
@fsoffel   faker has function to draw a random element from array

Reply:
Done.